providers/implementations/keymgmt/ml_kem_kmgmt.c.in

```diff
@@ -612,6 +618,36 @@ static int ml_kem_get_params(void *vkey, OSSL_PARAM params[])
                                   &ossl_ml_kem_encode_seed))
             return 0;
     }
+
```

Comment:
@levitte Can you comment on if this is the correct place to be putting this code?